src/theme/core.scss

```
@@ -80,6 +80,15 @@ ion-card-content {
 //Projects list
 .projects-list {
     padding-top: 0;
+    //Override Ionic's slotted avatar size (40x40 in MD, 36x36 in iOS) so the
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)